src/tools/gophish_export.py

```diff
@@ -253,6 +253,86 @@ def get_application(rawEvent):
     return application
 
 
+def find_unique_target_clicks_count(clicks):
+    """Return the number of unique clicks in a click dict."""
+    uniq_list = list()
+    for click in clicks:
+        if click["user"] not in uniq_list:
+            uniq_list.append(click["user"])
+    return len(uniq_list)
+
+
+def write_campaign_summary(api, assessment_id):
+    """Output a campaign summary report to JSON, console, and a text file."""
+    campaign_ids = get_campaign_ids(api, assessment_id)
+    campaign_data_template = "campaign_data.json"
+    campaign_summary_json = assessment_id + "_campaign_data.json"
+    campaign_summary_textfile = (
+        assessment_id + "_summary_" + str(datetime.now()) + ".txt"
+    )
+
+    with open(campaign_data_template) as template:
+        campaign_data = json.load(template)
+
+    logging.info("Writing campaign summary report to %s" % campaign_summary_textfile)
+    file_out = open(campaign_summary_textfile, "w+")
+    file_out.write("Campaigns for Assessment: " + assessment_id)
+
+    for campaign_id in campaign_ids:
+        campaign = api.campaigns.get(campaign_id)
+        if campaign.name.endswith("_level-1"):
+            level = "level-1"
+        elif campaign.name.endswith("_level-2"):
+            level = "level-2"
+        elif campaign.name.endswith("_level-3"):
+            level = "level-3"
+        elif campaign.name.endswith("_level-4"):
+            level = "level-4"
+        elif campaign.name.endswith("_level-5"):
+            level = "level-5"
+        elif campaign.name.endswith("_level-6"):
+            level = "level-6"
+        else:
+            continue
+
+        logging.info(level)
+        clicks = get_click_data(api, campaign_id)
+
+        total_clicks = api.campaigns.summary(campaign_id=campaign_id).stats.clicked
+        unique_clicks = find_unique_target_clicks_count(clicks)
+        if total_clicks > 0:
+            percent_clicks = unique_clicks / float(total_clicks)
+        else:
+            percent_clicks = 0
```

Comment:
Ensure `percent_clicks` is always a `float`:
```suggestion
            percent_clicks = 0.0
```

Reply:
See 5816a24f37e9a600f2b55946de18b7acb8d0da07